src/components/SamlProviderConfiguration/SamlProviderDataForm.jsx

```diff
@@ -28,10 +31,13 @@ class SamlProviderDataForm extends React.Component {
    */
   validateProviderDataForm = (formData, requiredFields) => {
     const invalidFields = requiredFields
-      .filter(field => !formData.get(field))
-      .reduce((prevFields, currField) => ({ ...prevFields, [currField]: true }), {});
+      .filter((field) => !formData.get(field))
+      .reduce(
+        (prevFields, currField) => ({ ...prevFields, [currField]: true }),
+        {},
+      );
```

Comment:
That's O(n^2). There is a better way
```suggestion
     const invalidFields = {};
     for (let field of requiredFields) {
        if (!formData.get(field)) invalidFields[field] = true;
     }
```